src/order/filter/zero_and_bounded_at_filter.lean

```diff
@@ -25,29 +25,33 @@ variables {α β : Type*}
 
 open_locale topological_space
 
-/-- A function `f : α → β` is `zero_at_filter` if in the limit it is zero. -/
+/-- If `l` is a filter on `α`, then a function `f : α → β` is `zero_at_filter l`
+  if it tends to zero along `l`. -/
 def zero_at_filter [has_zero β] [topological_space β] (l : filter α) (f : α → β) : Prop :=
 filter.tendsto f l (𝓝 0)
 
 lemma zero_is_zero_at_filter [has_zero β] [topological_space β] (l : filter α) : zero_at_filter l
   (0 : α → β) := tendsto_const_nhds
 
-/-- The submodule of funtions that are `zero_at_filter`. -/
+/-- `zero_at_filter_submodule l` is the submodule of `f : α → β` which
+tend to zero along `l`. -/
 def zero_at_filter_submodule [topological_space β] [semiring β]
   [has_continuous_add β] [has_continuous_mul β] (l : filter α) : submodule β (α → β) :=
 { carrier := zero_at_filter l,
   zero_mem' := zero_is_zero_at_filter l,
   add_mem' := by { intros a b ha hb, simpa using ha.add hb, },
   smul_mem' := by { intros c f hf, simpa using hf.const_mul c }, }
 
-/-- The additive submonoid of funtions that are `zero_at_filter`. -/
+/-- `zero_at_filter_add_submonoid l` is the additive submonoid of `f : α → β`
+which tend to zero along `l`. -/
 def zero_at_filter_add_submonoid [topological_space β]
   [add_zero_class β] [has_continuous_add β] (l : filter α) : add_submonoid (α → β) :=
 { carrier := zero_at_filter l,
   add_mem' := by { intros a b ha hb, simpa using ha.add hb },
   zero_mem' := zero_is_zero_at_filter l, }
 
-/-- A function `f: α → β` is `bounded_at_filter` if `f =O[l] 1`. -/
+/-- If `l` is a filter on `α`, then a function `f: α → β` is `bounded_at_filter l`
+if `f =O[l] 1`. -/
```

Comment:
Is that valid notation? Is so, why is it not used in the actual code?

Reply:
I don't know, that wasn't a part I wrote.

Reply:
Fair enough.